battleship-client/src/main/java/com/academy/solid/nie/client/ui/GameScene.java

```diff
@@ -72,7 +77,7 @@ private void handlePlayersMove(final Cell cell) {
                     .withButtonWhoExitSystem().display();
             socketServer.sendGameOverToOpponent();
         }
-        if (enemyBoard.isShipSunken(cell.getShip())){
+        if (enemyBoard.isShipSunken(cell.getShip())) {
```

Comment:
What would be needed to make this `ship.isSunken` instead? Feel free to answer me later.